config/fastly/snippets/check_for_remember_user_token_in_cookie_to_set_header.vcl

```diff
@@ -0,0 +1,19 @@
+sub vcl_recv {
```

Comment:
I don't understand why we have this script at all, we never use this header upstream...

Reply:
I think @benhalpern might know. IIRC, it was there to address some quirk...maybe with something on how we're doing shielding?